src/components/select/multi-select/multi-select-test.stories.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import { MultiSelect, Option } from "..";
+import partialAction from "../../../__internal__/utils/storybook/partial-action";
```

Comment:
same comment on this file as on the filterable-select one

Reply:
are you sure? It displays under `Select/MultiSelect/Test` for me:
![image](https://user-images.githubusercontent.com/22885392/206137432-16036291-af38-4016-841c-133b8a417f78.png)